openssl-dynamic/src/main/c/sslcontext.c

```diff
@@ -593,6 +593,25 @@ TCN_IMPLEMENT_CALL(jboolean, SSLContext, setCACertificate)(TCN_STDARGS,
     return rv;
 }
 
+TCN_IMPLEMENT_CALL(jboolean, SSLContext, setCACertificateBio)(TCN_STDARGS,
```

Comment:
It doesn't look like the return value is used ... should we just make this method void and throw if something goes wrong?